### ISSUE TITLE
[makeotf] NameError: global name 'familyFilePath' is not defined

### ISSUE THREAD

**Issue by mashabow** [adobe-type-tools/afdko#158]
I am getting `NameError` while compling a CID-keyed font:

```
Traceback (most recent call last):
  File "/Users/mashabow/bin/FDK/Tools/osx/../SharedData/FDKScripts/MakeOTF.py", line 2561, in <module>
    main()
  File "/Users/mashabow/bin/FDK/Tools/osx/../SharedData/FDKScripts/MakeOTF.py", line 2553, in main
    setMissingParams(makeOTFParams)
  File "/Users/mashabow/bin/FDK/Tools/osx/../SharedData/FDKScripts/MakeOTF.py", line 1625, in setMissingParams
    foundVert = checkIfVertInFeature(featPath)
  File "/Users/mashabow/bin/FDK/Tools/osx/../SharedData/FDKScripts/MakeOTF.py", line 1440, in checkIfVertInFeature
    fdir = os.path.dirname(familyFilePath)
NameError: global name 'familyFilePath' is not defined
```

[MakeOTF.py#L1440](https://github.com/adobe-type-tools/afdko/blob/2c3cbf35b05557179e72d36f6fc28eabcd1427c5/FDK/Tools/SharedData/FDKScripts/MakeOTF.py#L1440)

Maybe this is a typo for `featurePath`?

**Comment by readroberts** [comment]
You are correct. You can fix this on your local system by simply changing familyFilePath to "featurePath". There is also another bug. This function makes two attempts to find the included feature file. The first is relative to the current working directory; the second is relative to the parent feature file. The order should be reversed. I will fix this.

**Comment by LiXiangchen** [comment]
I posted a fix for the code #149 , looking forward to official fix.

**Comment by readroberts** [comment]
Fixed in commit 76ae0849ec197e240a12a4157c7aac9ca4375151.
